### ISSUE TITLE
Your system folder path does not appear to be set correctly. Please open the following file and correct this: index.php

### ISSUE THREAD

**Issue by DeepRootedCoder** [opensourcepos/opensourcepos#2577]
### Background information

__IMPORTANT: If you choose to ignore this issue report template, your issue will be closed as we cannot help without the requested information.__

Please make sure you tick (add an x between the square brackets with no spaces) the following check boxes:

- [x] Reporting an issue of an unmodified OSPOS installation
- [x] Checked [open and closed issues](https://github.com/opensourcepos/opensourcepos/issues?utf8=%E2%9C%93&q=is%3Aissue) and no similar issue was already reported (please make sure you searched!)
- [x] Read [README](https://github.com/opensourcepos/opensourcepos/blob/master/README.md), [WHATS_NEW](https://github.com/opensourcepos/opensourcepos/blob/master/WHATS_NEW.txt), [INSTALL.md](https://github.com/opensourcepos/opensourcepos/blob/master/INSTALL.md) and [UPGRADE](https://github.com/opensourcepos/opensourcepos/blob/master/UPGRADE.txt)
- [x] Read the [FAQ](https://github.com/opensourcepos/opensourcepos#faq) for any known install and/or upgrade gotchas (in specific PHP extensions installed)
- [x] Read the [wiki](https://github.com/opensourcepos/opensourcepos/wiki)
- [x] Executed any database upgrade scripts if an upgrade pre 3.0.0 (e.g. database/2.4_to_3.0.sql)
- [x] Aware the installation code is in [bintray](https://bintray.com/jekkos/opensourcepos/opensourcepos/view/files?sort=updated&order=asc#files) (see README), and [GitHub master](https://github.com/opensourcepos/opensourcepos/tree/master) is for [developers only](https://github.com/opensourcepos/opensourcepos/wiki/Development-setup) and therefore not complete nor stable

### Installation information

- OSPOS version is: 3.3.0
- OSPOS git commit hash is:  
- PHP version is: 7.3
- MySQL or MariaDB version is: 10.0 
- OS and version is: Raspian OS Buster - Debian 10
- WebServer is: Nginx + Adminer 4.2.5
- Selected language is: 
- (If applicable) Docker installation: 
- (If applicable) Installation package for the LAMP/LEMP stack is:  LEMP

### Issue / Bug  / Question / New Feature

Please write your issue here. If a bug, please make sure to provide as much information as possible including configuration settings (e.g. Decimals set, Tax mode), language and steps to reproduce the bug.

Using Raspberry Pi 3B+

I'm getting the following error when I access localhost/opensourcepos/public:
Your system folder path does not appear to be set correctly. Please open the following file and correct this: index.php
Nginx, PHP and Adminer are working fine. However I am using PHP 7.3. The composer.json file contains "php": "^5.6 || ^7.0". I'm not sure if it'd make a difference but that's something I noticed.
I have tried Apache 2.4 and phpMyAdmin with MySQL too but have encountered the same issue. 
Due to the obsolete PHP 7.0 modules and MySQL packages I had to switch to PHP 7.3 and MariaDB. I faced many issues but resolved them, I will share them to ease the future installations for raspberry pi. However I can't resolve this issue.
Thanks in advance.

**Comment by daN4cat** [comment]
Have you downloaded the zip file or cloned GitHub repo?

**Comment by daN4cat** [comment]
Btw it's tested with PHP7.3 and works fine.

**Comment by DeepRootedCoder** [comment]
Yes, I downloaded the zip file from the releases section. It was around 23.5MB.

**Comment by daN4cat** [comment]
Have you checked the wiki page about RaspberryPi installation?

**Comment by DeepRootedCoder** [comment]
Yes. I have followed the exact instructions on the wiki page of raspberry pi around 10 times. The wiki page however hasn't been updated for quite some time. So maybe there are some requirements or configurations that I need to do to make it work with updated packages.

**Comment by daN4cat** [comment]
Probably, I wanted to make sure we removed the obvious.

Since you use nginx instead of Apache there might be something with that but I didn't install OSPOS on nginx to help.
I used nginx as reverse proxy only, which is a different story.

**Comment by DeepRootedCoder** [comment]
Hmm... I'll try downgrading to Raspbian Jessie and try to work with older packages and see what happens.
Thanks for your help.

**Comment by daN4cat** [comment]
Let us know how it goes, but looks like be an nginx configuration issue.

**Comment by DeepRootedCoder** [comment]
Okay, so I finally got it working!

Current Setup:
Raspberry Pi 3B+
PHP 7.3
Raspbian Buster (Debian 10)
Nginx 
OSPOS v3.3.0
Mariadb 10.3

I was getting a HTTP error 500 on Nginx.

***Changes***

-Manually created 'admin' user in adminer.
-HTTP 500 errors not showing up in the Nginx errorlog file:
Changes in file: /var/www/html/opensourcepos/application/config/database.php
Line 85:  ENVIRONMENT !== 'development'

Changes in file: /var/www/html/opensourcepos/public/index.php
Line 56: changed 'production' to 'development'
Line 69: changed error_reporting to error_reporting(E_ALL);

-Now, Nginx started to report errors in the log. I was getting an error (function not defined) on line 42 in directory: /opensourcepos/application/hooks/load_config.php
So, I commented the following line:
Line 42: bcscale(max(2, totals_decimals() + tax_decimals()));

OSPOS started working fine after that. I hope you guys can look into it.
I wish to create an updated installation manual for future Raspberry Pi installations,that is if you guys allow it.

Thanks for all your help! @daN4cat
